st2common/st2common/hooks.py

```diff
@@ -106,24 +104,5 @@ def _validate_token(request):
 
         token_in_headers = headers.get('X_Auth_Token', None)
```

Comment:
can they be the same strings? are there some weird restrictions that would prevent doing so.

Ideally, X-Auth-Token